### PR TITLE
GitHub get path data command

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
For Update Docker Image Auto, added this command to retrieve data of given path. This will be used to get SHA of files for docker image update playbook tasks.
this command was needed because currently docker image playbook is implemented retrieving SHA of files using un-authenticated http calls, which has a small quota. This command solves this by performing authenticated call (using integration params) so quota will not be reached (authenticated has 1k + calls per hour allowed)

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 1]
```diff
@@ -1834,6 +1834,56 @@ def fetch_incidents_command_rec(start_time, last_time, page=1):
...
+    repo: str = args.get('repository') or REPOSITORY
+    organization: str = args.get('organization') or USER
+    branch_name: Optional[str] = args.get('branch_name')
+
+    url_suffix = f'/repos/{organization}/{repo}/contents/{relative_path}'
+    url_suffix = f'{url_suffix}?ref={branch_name}' if branch_name else url_suffix
+
+    headers = {
+        'Authorization': "Bearer " + TOKEN,
+        'Accept': 'application/vnd.github.VERSION.object',
```
Comment (Shellyber) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1855]: Why do we add this? is this needed? 
```suggestion
        'Accept': 'application/vnd.github.VERSION.object',
```

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 2]
```diff
@@ -1834,6 +1834,56 @@ def fetch_incidents_command_rec(start_time, last_time, page=1):
     return incidents, last_time
 
 
+def get_path_data():
```
Comment (Shellyber) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1837]: Shouldn't be `get_file_data`?

**Review by Shellyber — COMMENTED** [review]
Looking good! 👍 
Please see my comments

File: Packs/GitHub/Integrations/GitHub/GitHub.py (hunk 1 above)
Comment (tomneeman151293) [reply Packs/GitHub/Integrations/GitHub/GitHub.py:1855]: For consistency, object returns the expected. I am afraid in future they will change their API to do something else if Accept was not given.

File: Packs/GitHub/Integrations/GitHub/GitHub.py (hunk 2 above)
Comment (tomneeman151293) [reply Packs/GitHub/Integrations/GitHub/GitHub.py:1837]: It works for path as well. Not only file so I included it

Comment (tomneeman151293) [reply Packs/GitHub/Integrations/GitHub/GitHub.py:1837]: See the README example!

**Review by Shellyber — APPROVED** [review]

**Pull request merged**